packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -221,3 +221,7 @@
 	fill: currentColor;
 }
 
+.edit-site-global-styles-screen-root__active-style-tile,
+.edit-site-global-styles-screen-root__active-style-tile-preview {
+	border-radius: $radius-small !important; // Todo: revisit use of Card component and/or remove override
```

Comment:
is the `!important` necessary?

Reply:
The component styles are added inline, so unless there's a better way to perform the override (?) I think it is necessary.

Reply:
I don't see inline styles on those components — we can still apply overrides with a little specificity hack that is not as invasive as `!important`:

```scss
.edit-site-global-styles-screen-root__active-style-tile {
	// Todo: revisit use of Card component and/or remove override
	// The &#{&} is a workaround for the specificity of the Card component.
	&#{&},
	&#{&} .edit-site-global-styles-screen-root__active-style-tile-preview {
		border-radius: $radius-small;
	}
}
```

Reply:
Sorry I meant added by js (I think). TIL there's a way to override that :)